docker/safety-rules/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM debian:buster AS toolchain
+
+# To use http/https proxy while building, use:
+# docker build --build-arg https_proxy=http://fwdproxy:8080 --build-arg http_proxy=http://fwdproxy:8080
+
+RUN apt-get update && apt-get install -y cmake curl clang git
+
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain none
+ENV PATH "$PATH:/root/.cargo/bin"
+
+WORKDIR /libra
+COPY rust-toolchain /libra/rust-toolchain
+RUN rustup install $(cat rust-toolchain)
+
+FROM toolchain AS builder
+
+COPY . /libra
+
+RUN cargo build --release -p libra-node -p cli -p config-builder -p safety-rules && cd target/release && rm -r build deps incremental
+
+### Production Image ###
+FROM debian:buster AS prod
+
+RUN addgroup --system --gid 6180 libra && adduser --system --ingroup libra --no-create-home --uid 6180 libra
+
+RUN mkdir -p /opt/libra/bin
+COPY --from=builder /libra/target/release/safety-rules /opt/libra/bin
```

Comment:
Is this the correct binary and built target?

Reply:
Excellent!

Reply:
Yes!